config/configgrpc/gzip.go

```diff
@@ -15,6 +15,6 @@
 package configgrpc
 
 import (
-	// import the gzip package with auto-registers the gzip grpc compressor
+	// Import the gzip package with auto-registers the gzip gRPC compressor.
```

Comment:
```suggestion
	// Import the gzip package which auto-registers the gzip gRPC compressor.
```

Reply:
Thanks for catching this!! Addressed the other comments as well :D